composio/cli/add.py

```diff
@@ -174,6 +174,7 @@ def add_integration(
     else:
         connection = entity.initiate_connection(
             app_name=name,
+            auth_mode=auth_modes[0],
```

Comment:
@angrybayblade was working on the same with a board support for giving user access to which auth mode should be picked first (`OAUTH2`, `OAUTH1`). Just leaving this here as a comment, so you're aware of this change